### ISSUE TITLE
Windows Support

### ISSUE THREAD

**Issue by synapticarbors** [conda-forge/xgboost-feedstock#3]
It looks like xgboost supports windows:

https://xgboost.readthedocs.io/en/latest/build.html#building-on-windows

It would be nice to have a package built for windows even though one is available on `defaults`. It might be worth downloading the tarball from anaconda.org and looking at the recipe that's included for guidance:

https://anaconda.org/anaconda/py-xgboost/files

**Comment by beckermr** [comment]
Yeah def! I have been waiting for them to release a new package before working more on this. It has been almost a year since they put anything out. ☹️

**Comment by beckermr** [comment]
"This" being this recipe in general. Multithreading is also not supported on OSX.

**Comment by synapticarbors** [comment]
Ok, great. I've just been looking at the feedstock as I try to put together a recipe for LightGBM (https://github.com/conda-forge/staged-recipes/pull/3257). The docs for that package says it requires openmp explicitly, so I'm waiting on the PR that fixes that too.

**Comment by beckermr** [comment]
Cool. You can check out my open PR for getting openmp to work on a mac. Maybe this will help with LightGBM? You can use the conda gcc and libgcc packages. For xgboost I had to patch the makefiles more.

**Comment by jakirkham** [comment]
Now that we are building with CMake ( https://github.com/conda-forge/xgboost-feedstock/pull/28 ), maybe this is more approachable?

**Comment by jakirkham** [comment]
Done in PR ( https://github.com/conda-forge/xgboost-feedstock/pull/60 )
